.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Thank you for your contribution to the Graph Data Science Client project.
+
+<!-- Please include a summary of the change, such as, which issue was fixed or feature was added. 
+If relevant, link to the corresponding issue.
+Please also include relevant motivation and context. List any dependencies that are required for this change. -->
+
+Before submitting this PR, please make sure:
+- [ ] You signed the [Neo4j CLA](https://neo4j.com/developer/cla/#sign-cla) (Contributor License Agreement) so that we are allowed to ship your code in our library
```

Comment:
All the details are outlined in `CONTRIBUTING.md` so we might want to refer to that?

Reply:
added a reference :+1: